libbeat/docs/release-notes/highlights/highlights-7.6.0.asciidoc

```diff
@@ -18,8 +18,58 @@ For a list of bug fixes and other changes, see the {beats}
 
 // tag::notable-highlights[]
 
-//[float]
-//==== Highlight
+[float]
+==== Support added for TLS 1.3
+
+In 7.6, we've added support for TLS 1.3, enabling you to take advantage
+of the security and performance improvements available in TLS 1.3 when {beats}
+connects to an {es} cluster that supports it.
+
+//REVIEWERS: Any more details to add here. GitHub issues/PRs were spare.
+
+[float]
+==== Azure cloud monitoring improvements
+
+The new {metricbeat-ref}/metricbeat-metricset-azure-storage.html[`storage`]
+metricset in the {metricbeat} {metricbeat-ref}/metricbeat-module-azure.html[Azure]
+module adds the ability to collect metrics from storage accounts.
+
+We've also made it easier for you to do a cost analysis on metrics
+collection by adding a cost warning message to each metrics API call. The
+message indicates the cost applied while retrieving metric values from Azure
+resources.
+
+[float]
+==== Google cloud improvements
+
+Starting in 7.6, you can use {functionbeat} to get events from 
+https://cloud.google.com/pubsub/[Google Pub/Sub] and
+https://cloud.google.com/storage/[Google Cloud Storage].
+
+You define functions that respond to Google triggers then export them as YAML
+files that you deploy by using the Google Cloud Deployment Manager. 
+
+// REVIEWERS: I can't say much about this yet because I just started working on
+// the docs. Please let me know what details we should include here and how we
+// want to position this.
```

Comment:
@kvch Can you provide some details please? 

Some other GCP features/improvements that made it into 7.6:
- https://github.com/elastic/beats/pull/14829
- https://github.com/elastic/beats/pull/15200
- https://github.com/elastic/beats/pull/14553

Reply:
awesome! Thanks for the links. On Monday, I'll beef up the google cloud section to include the other improvements!

Reply:
@kvch I see there's a little blurb in the Logging blog, so I'll just use that. Thanks!